src/styles/card.scss

```diff
@@ -0,0 +1,78 @@
+.card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  font-family: $bodyFont;
+  padding: 32px 16px 16px;
+  overflow: auto;
+  &__image {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto 40px;
+  }
+  &__apple {
+    color: #060b35;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+  &__code {
+    color: #616070;
+    text-align: left;
+    margin-bottom: 16px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+  &__button {
+    color: #fff;
+    text-align: center;
+    background-color: #00acdc;
+    border-radius: 5px;
+    width: 100%;
+    height: 40px;
+    font-family: $bodyFont;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    border: 1px solid #00acdc;
+  }
+  &__button:hover {
+    background-color: white;
+    color: #00acdc;
+    border: 1px solid #00acdc;
```

Comment:
The `border` property is duplicated for the `.card__button` and `.card__button:hover` at lines 39 and 44. Since the border color and width remain the same on hover, you can omit the border property from the hover state.